tests/HistoryRepositoryTest.php

```diff
@@ -51,18 +51,46 @@ public function testCreateExtraFieldsBD()
         $this->assertEquals($this->getJSONFixture('create_success_history_state.json'), $result);
     }
 
-    public function testCreateNotAllFields()
+    public function testCreateNotAllFieldsRus()
     {
-        $this->assertExceptionThrowed(CreateWithoutRequiredFieldsException::class, 'One of required fields does not filled.', function () {
+        $data = '[localization]' . PHP_EOL . 'locale = ru' . PHP_EOL;
+
+        file_put_contents("{$this->iniDirName}/settings.ini", $data);
```

Comment:
let's move it to the separate function `setLocale` in the parent base test class and use it here and in CalculatorServicetest